.npmignore

```diff
@@ -1,3 +1,9 @@
 /.tern-port
 /test
 /local
+/bin
```

Comment:
Sorry about that.  Should the I exclude `generate-identifier-regex.js` and `update_authors.sh` or leave the directory alone?